.genx/templates/entityManager.hbs

```diff
@@ -1,4 +1,7 @@
 <entity-management
+    enable-row-flashing
+    async-add
```

Comment:
row flashing should work for async and non async transaction so I would remove `async-add` attribute. here is the [source](https://github.com/genesislcap/foundation-ui/blob/master/packages/foundation/foundation-marketplace/grids/grid-pro/src/datasource/base.datasource.ts#L292-L302)